src/mail-app/mail/view/MailViewerToolbar.ts

```diff
@@ -22,34 +22,35 @@ import { styles } from "../../../common/gui/styles"
 export interface MailViewerToolbarAttrs {
 	mailboxModel: MailboxModel
 	mailModel: MailModel
-	mailViewerViewModel?: MailViewerViewModel
 	selectedMails: Mail[]
-	selectNone?: () => void
 	actionableMails: () => Promise<readonly Mail[]>
+	primaryMailViewerViewModel?: MailViewerViewModel
```

Comment:
my comment got lost but I just did find it very confusing to have two different viewModels. I know we have the same for mails and it's also not great tbh but there it is kind of more understandable. We should probably just let `MailViewModel` to decide on all these things.

It's okay if we merge it like this and refactor later but I am kind of tempted to get rid of the whole actionable thing soon.